samples/azure-communication-chat-samples/src/androidTest/java/com/azure/azure_communication_chat_samples/CreateChatThread.java

```diff
@@ -0,0 +1,106 @@
+package com.azure.azure_communication_chat_samples;
```

Comment:
I'm not a big fan of using underscored in our package names, not sure if this would be a good name. Any thoughts @anuchandy?
```suggestion
package com.azure.samples.communication.chat;
```